.travis.yml

```diff
@@ -38,6 +39,11 @@ install:
       cargo install hyperfine
     fi
     hyperfine --version
+    if [ ! $(which strace) ]; then
+      if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+        sudo apt-get install strace
```

Comment:
strace isn’t there by default ?

Reply:
When I tested on my Fedora machine, it seems that `strace` is not included by default

Reply:
Please test on travis.

Reply:
Sure. I'll see the effect if I remove these lines